08_cell_line_prediction/download_drug_data.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>What is the difference between <code>0</code> and <code>NaN</code> in  this matrix? Is <code>0</code> a known non-responder where  <code>NaN</code> doesn't have any data? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Yep, exactly - `NaN` are cell lines that weren't assayed for the relevant drug, `0` are cell lines that were assayed and were resistant/non-responders. I'll add a comment.